pkg/helm/handlers/handlers.go

```diff
@@ -143,3 +149,70 @@ func (h *helmHandlers) HandleChartGet(user *auth.User, w http.ResponseWriter, r
 	res, _ := json.Marshal(resp)
 	w.Write(res)
 }
+
+
+func (h *helmHandlers) HandleHelmUpgradeRelease(user *auth.User, w http.ResponseWriter, r *http.Request) {
+	var req HelmRequest
+
+	err := json.NewDecoder(r.Body).Decode(&req)
+	if err != nil {
+		serverutils.SendResponse(w, http.StatusBadGateway, serverutils.ApiError{Err: fmt.Sprintf("Failed to parse request: %v", err)})
```

Comment:
If we cannot parse, then bad request should be more appropriate?

Reply:
Done